pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -276,9 +277,9 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 
 	glog.Infof("Creating device plugin handler: %t", devicePluginEnabled)
 	if devicePluginEnabled {
-		cm.devicePluginHandler, err = deviceplugin.NewHandlerImpl(updateDeviceCapacityFunc)
+		cm.devicePluginManager, err = deviceplugin.NewManagerImpl(updateDeviceCapacityFunc, pluginapi.KubeletSocket)
```

Comment:
Perhaps we can avoid the dependency on pluginapi at cm layer by having a private constructor newManagerImpl and have deviceplugin.NewManagerImpl call it with pluginapi.KubeletSocket socketname?

Reply:
Cool - updating PR in a sec with that change.